README.md

```diff
@@ -20,7 +20,11 @@ Add the following dependency to your `build.gradle`
 implementation 'com.grafana:grafana-opentelemetry-starter:1.0.1'
 ```
 
-Next, register the OpenTelemetry logback appender in `logback-spring.xml` (or `logback.xml`):
+The next step depends on your logging framework - logback and log4j2 are supported: 
+
```

Comment:
Maybe let's add a `Logging` header and then have subsections `Logback` and `Log4j2` under it. 

### Logging
#### Logback
#### Log4j2